### PR TITLE
docs: Add docs for SSH Keys and Cloud Provider Credentials

### PR DESCRIPTION
**Pull request by jfoliveira**
- [x] Add documentation basic guidelines
- [x] docs: Add a Cloud Provider Credential
- [x] docs: Remove a Cloud Provider Credential
- [x] docs: Add home page for Cloud Provider Credentials docs
- [x] docs: Add an SSH Key
- [x] docs: Remove an SSH Key
- [x] docs: Add home page for SSH Keys docs

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -5,3 +5,18 @@ Make sure all files created in this folder adhere to the following basic rules:
...
+## What to document?
+We should document use cases that will help our users on achieve success on their tasks while using Devopness platform.
+
+## How to document?
+Step by step. Each step should be clear, with a short sentence.
+If a process is long, break it down into multiple small steps.
+That will help the reader to understand each step without pressure, then breath, then move on to the next step.
+
+## What should NOT be included in the documentation?
+1. Do not go to deep into business rules that are already validated and communicated by the API
```
Comment (renan-bg) [inline docs/README.md:19]:
```suggestion
1. Do not go too deep into business rules that are already validated and communicated by the API
```

File: docs/README.md [hunk 2]
```diff
@@ -5,3 +5,18 @@ Make sure all files created in this folder adhere to the following basic rules:
...
+As Devopness product is built with an API-first approach and also we thrive for keeping API responses and validation messages easy to be understood by machines and humans alike, there's no need to repeat ourselves in the docs explaining which field values are valid or not.
```
Comment (renan-bg) [inline docs/README.md:20]:
```suggestion
as Devopness product is built with an API-first approach. We strive for keeping API responses and validation messages easily understood by machines and humans alike, there's no need to repeat ourselves in the docs explaining which field values are valid or not.
```

File: docs/README.md [hunk 3]
```diff
@@ -5,3 +5,18 @@ Make sure all files created in this folder adhere to the following basic rules:
...
+It is Devopness API responsibility to communicate the validation rules to the end users in a clear way, so that should not require extra documentation to explain API messages.
+- We might, however, want to produce `Overview` or `Deep dive` articles to conceptually guide users through complex use cases, but that falls beyond the traditional use cases step by step approach.
```
Comment (renan-bg) [inline docs/README.md:22]:
```suggestion
- We might, however, want to produce `Overview` or `Deep dive` articles to conceptually guide users through complex use cases, but that falls beyond the traditional step by step approach.
```

File: docs/cloud-providers/remove-cloud-providers-credentials.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: Remove a Cloud Provider Credential
+intro: When your Cloud Provider Credentials are rotated or you don;t need to support a given cloud provider account you can remove from Devopness the credentials that are no longer being used.
```
Comment (renan-bg) [inline docs/cloud-providers/remove-cloud-providers-credentials.md:3]:
```suggestion
intro: When your Cloud Provider Credentials are rotated or you don't need to support a given cloud provider account you can remove from Devopness the credentials that are no longer being used.
```

File: docs/README.md (hunk 3 above)
Comment (jfoliveira) [reply docs/README.md:22]: @renan-bg what about?
```suggestion
- We might, however, want to produce `Overview` or `Deep dive` articles to conceptually give users detailed explanation on specific topics that fall beyond the more common use-case-based step by step.
```

File: docs/ssh-keys/remove-ssh-key.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title: Remove an SSH key
```
Comment (mlpbraga) [inline docs/ssh-keys/remove-ssh-key.md:2]:
```suggestion
title: Remove an SSH key
intro: Sometimes you may need to remove a SSH key refered to a former member of your team or to a machine that is no longer available. Remove this SSH keys to ensure your server safety. 
```

**Review by plantaz — COMMENTED** [review]
Suggestions of possible improvements

File: docs/cloud-providers/add-cloud-providers-credentials.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        - /docs/cloud-providers/remove-cloud-providers-credentials
+    featured:
+---
+
+# Add a Cloud Provider Credential
+1. In the upper-right corner of any page, click your profile icon then click `Cloud Providers`
+    - A list of existing Cloud Providers Credentials will be displayed
+2. On the upper-right corner of the list click `ADD CLOUD PROVIDER`
+3. Use the drop-down menu to choose a Cloud Provider
+4. Provide a name to the `Cloud Provider Credential` being added. For example, "Company Laptop"
```
[the remark this answers is not in the document]
Comment (jfoliveira) [reply docs/cloud-providers/add-cloud-providers-credentials.md:21]: @mlpbraga thanks! what about?
```suggestion
4. Provide a name to your new `Cloud Provider Credential`. For example, "Test Account"
```

File: docs/ssh-keys/remove-ssh-key.md (hunk 5 above)
Comment (jfoliveira) [reply docs/ssh-keys/remove-ssh-key.md:2]: @mlpbraga perhaps ...?
```suggestion
title: Remove an SSH key
	intro: Sometimes you may need to remove an SSH key related to a former member of your team or to a machine that is no longer available. Remove these SSH keys to ensure your servers remain only accessible by trusted sources.
```

File: docs/ssh-keys/add-ssh-key.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+---
+title: Add an SSH key
+intro: Sometimes you and your team members need to connect to your servers to run commands from the terminal. Add SSH keys to environment servers to ensure secure server access by only the authorized SSH keys.
```
[the remark this answers is not in the document]
Comment (jfoliveira) [reply docs/ssh-keys/add-ssh-key.md:3]: @mlpbraga sometimes the user just need terminal access to run a single command. It seems to me using `manage` here might too much. What do you think?

File: docs/ssh-keys/remove-ssh-key.md (hunk 5 above)
Comment (mlpbraga) [reply docs/ssh-keys/remove-ssh-key.md:2]: perfect!

File: docs/ssh-keys/add-ssh-key.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (mlpbraga) [reply docs/ssh-keys/add-ssh-key.md:3]: makes sense to me

**Review by mlpbraga — APPROVED** [review]

File: docs/ssh-keys/remove-ssh-key.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+	intro: Sometimes you may need to remove an SSH key related to a former member of your team or to a machine that is no longer available. Remove these SSH keys to ensure your servers remain only accessible by trusted sources.
```
Comment (Silva97) [inline docs/ssh-keys/remove-ssh-key.md:3]:
```suggestion
intro: Sometimes you may need to remove an SSH key related to a former member of your team or to a machine that is no longer available. Remove these SSH keys to ensure your servers remain only accessible by trusted sources.
```

**Review by Silva97 — APPROVED** [review]

**Pull request merged**